libs/domain/product/images/src/images.component.ts

```diff
@@ -85,8 +89,18 @@ export class ProductImagesComponent extends ProductMixin(
 
     return html`<oryx-layout
       class="main"
-      layout=${ifDefined(imageLayout)}
       behavior=${ifDefined(scrollBehavior)}
+      layout=${ifDefined(imageLayout)}
+      .options=${{
+        rules: [
+          {
+            layout: {
+              showArrows: false,
```

Comment:
should we pass `false` here?

Reply:
yes. enabling the arrows on images takes another ticket as there are a few edge cases that did not work.